### DISCUSSION TITLE
Images as Eye shape

### DISCUSSION THREAD

**Discussion by Rayane-K** [dagronf/QRCode#53] [Ideas]
Hello !
Could it be possible to put images (the same way it's possible to put images as Data in LogoTemplate and onPixel and offPixel...) for EyeShapes ?

I would love to put some custom images as eyes, but it's not possible at the moment.

What do you think ?

**Comment by dagronf** [comment]
@Rayane-K thanks for the feedback!  Just wondering, how would the eye image work?  Could you draw a quick representation of what you're thinking of?

Because the eye shape consists of two parts (which this library calls the 'eye' and the 'pupil') and these are both really important to the legibility for the qrcode - would the image replace the entire eye component? Or are you thinking its like a new fill style for the pupils?

**Comment by Rayane-K** [comment]
![IMG_0137](https://github.com/user-attachments/assets/ae4476e5-b60f-41cc-8134-5a4c0ed513ba)
![IMG_0138](https://github.com/user-attachments/assets/517e63d0-fac5-409a-9e4c-22968297037d)


Sorry for the late reply. Here is what I mean, I would like to put images to completely replace the eye component including the pupil.
